pom.xml

```diff
@@ -457,6 +458,11 @@
                     <artifactId>maven-checkstyle-plugin</artifactId>
                     <version>${maven.checkstyle.plugin.version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-release-plugin</artifactId>
+                    <version>3.0.0-M7</version>
```

Comment:
This should not be needed, since we inherit it from the FINOS Parent POM, see https://github.com/finos/legend-engine/blob/master/pom.xml#L23

Also note that 3.0.0-M7 breaks integration with maven-gpg-plugin, see https://github.com/apache/maven-release/pull/125#issuecomment-1302613138 ; better to stick with M5 version.

Reply:
yes, now that parent POM 5 is released (thank you), it's better to upgrade parent